pinot-spi/src/main/java/org/apache/pinot/spi/utils/CommonConstants.java

```diff
@@ -704,6 +708,9 @@ public enum Type {
     }
 
     public static final String PREFIX_OF_CONFIG_OF_PINOT_FS_FACTORY = "pinot.broker.storage.factory";
+
```

Comment:
nit: setting key is a bit unusual. 
It seems `pinto.broker.camelCaseSuffix` or `pinot.broker.dot.between.every.word` is more common.

Reply:
Yep, the dots approach seems to be the common one.

Changed to `pinot.broker.use.msqe.schema`

578fea9546

Reply:
Done!

266af9cb30